VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -385,6 +389,25 @@ public String getOAuthClientId() {
         return getString(OAUTH_CLIENT_ID);
     }
 
+    /**
+     * @return A list of URIs for updating the app, or an empty list if none are available.
```

Comment:
The token replacement is being done internally in the method, so a public javadoc wouldn't be an appropriate venue to document it, and the code is pretty self-evident. Nonetheless, I have added an inline comment explaining it in commit 27ae618bd9896badef5bd4dd600d6431b076ede7.